dashboard/modules/job/cli.py

```diff
@@ -321,6 +321,33 @@ def stop(address: Optional[str], no_wait: bool, job_id: str):
             time.sleep(1)
 
 
+@job_cli_group.command()
+@click.option(
+    "--address",
+    type=str,
+    default=None,
+    required=False,
+    help=(
+        "Address of the Ray cluster to connect to. Can also be specified "
+        "using the RAY_ADDRESS environment variable."
+    ),
+)
+@click.argument("job-id", type=str)
+@add_click_logging_options
+@PublicAPI(stability="alpha")
+def delete(address: Optional[str], job_id: str):
```

Comment:
If we want to rely on this API for job deletion, why don't we just support batch deletion? Otherwise, it is too inconvenient

Reply:
How should we specify a batch using the CLI?  Specifying all the job ids doesn't sound much easier than submitting a bunch of commands.

Maybe the recommended approach should be to use the SDK; it doesn't sound too inconvenient to programmatically delete a batch of jobs using the SDK (e.g. using a `for` loop).